components/clp-package-utils/clp_package_utils/controller.py

```
@@ -0,0 +1,638 @@
+            "CLP_REDUCER_UPSERT_INTERVAL": str(self._clp_config.reducer.upsert_interval),
+        }
+
+    def _set_up_env_for_webui(self, container_clp_config: CLPConfig) -> EnvVarsDict:
+        """
+        Sets up environment variables and settings for the Web UI component.
+
+        :param container_clp_config: CLP configuration inside the containers.
+        :return: Dictionary of environment variables necessary to launch the component.
+        """
+        component_name = WEBUI_COMPONENT_NAME
+        logger.info(f"Setting up environment for {component_name}...")
```

Ruff (0.14.0)

Logging statement uses f-string

(G004)